src/neo/Network/P2P/Payloads/OraclePayload.cs

```diff
@@ -48,20 +48,20 @@ public int Size
             }
         }
 
-        private UInt256 _hash = null;
-        public UInt256 Hash
+        private UInt160 _hash = null;
+        public UInt160 Hash
```

Comment:
Why 160? Just to reduce the size?

Reply:
We use UInt256 only for `IInventory`

Reply:
Maybe it should be a `IInventory`. What do you think @erikzhang ?

Reply:
I think it should be `IInventory`, which can reduce repeated sending, just like normal tx/block/consensus message. We use `inv` message to push new message.